spikingjelly/activation_based/triton_kernel/flex_sn_inductor/__init__.py

```
@@ -1,9 +1,29 @@
+        FlexSNScan,
+        eager_scan,
+        eager_scan_final_state,
+        lowerable_scan,
+        lowerable_scan_final_state,
+        lowerable_scan_available,
+        dynamo_hop_available,
+        lowerable_while_loop_scan,
+        lowerable_while_loop_scan_final_state,
+        lowerable_while_loop_available,
+    )
 except BaseException as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `BaseException`

(BLE001)